chalice/requirements.txt

```diff
@@ -0,0 +1,4 @@
+boto3==1.7.59
+psycopg2-binary==2.7.5
+peewee==3.6.4
+flake8==3.5.0
```

Comment:
in theory `flake` could stay just as a local dependency (we can have a `requirements-dev.txt` which includes `requirements.txt` and add this package too), so it doesn't have to be installed along with the lambda. We are not using all the package maximum size yet, but it's worth to keep it in mind.